Document how to change the MTU [CI SKIP] (#640)
harvester/docs: docs/vm/create-vm.md

## Volumes

> You can add one or more additional volumes via the `Volumes` tab, by default the first disk will be the `root disk`, you can change the boot order by dragging and dropping volumes, or using the arrow buttons.

Reviewer: ```suggestion
You can add volumes on the **Volumes** tab. By default, the first disk is used as the root disk. You can change the boot order by dragging and dropping volumes, or using the arrow buttons.
```
